### ISSUE TITLE
App hangs between post and action mailer

### ISSUE THREAD

**Issue by scroniser** [julianguyen/ifme#220]
If there is no internet connection, the app seems to 'hang' on the local install after post request is sent if action mailer isn't able to send. Unsure if there's a 'failsafe' action to catch the error - I'd suggest implementing if it isn't there.

**Comment by stellacotton** [comment]
Hey @scroniser, just want to make sure I'm understanding the issue:

1. Run the app locally (but without any email setup)
2. Execute an action that might trigger an email (like inviting someone to be an ally)
3. The app hangs indefinitely until the server times out

Does that sound similar to what you're seeing?

**Comment by scroniser** [comment]
@stellacotton Yep, that's what I'm trying to say!

**Comment by stellacotton** [comment]
In your application.yml are

`SEND_EMAIL: "false"`
`RAISE_DELIVERY_ERRORS: "false"`

set to false?

**Comment by scroniser** [comment]
Yes - it's supposed to be, right? I didn't change it from the original...?

**Comment by stellacotton** [comment]
Yep, sounds like this is a bug. Your local app shouldn't try to be sending email at all. What post request were you making?

**Comment by scroniser** [comment]
Posting a comment to someone else's 'moment'.

**Comment by BonMatts** [comment]
With the inclusion of the letter_opener gem, I could not replicate this behavior. I did, however, come across another error, in which a 500 was being served. The logs showed the message 
` ArgumentError (An SMTP From address is required to send a message. Set the message smtp_envelope_from, return_path, sender, or from address.)`

and letter opener was not opening the email in another tab. 
Apparently, letter opener does not work if there is not a "real" email address.  ([ref](https://github.com/ryanb/letter_opener/issues/1#issuecomment-6379461) )  

To fix this issue, I modified `development.env` to have a "real" email address. 
`SMTP_ADDRESS="test@test.com"`

Will submit a pull request with the modification in `development.example.env` shortly.
